src/SFA.DAS.EmployerAccounts.Api/Controllers/EmployerUserController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using System.Web.Mvc;
+using SFA.DAS.EmployerAccounts.Api.Attributes;
+using SFA.DAS.EmployerAccounts.Api.Orchestrators;
+
+namespace SFA.DAS.EmployerAccounts.Api.Controllers
+{
+    [System.Web.Http.RoutePrefix("api/user/{userRef}")]
+    public class EmployerUserController : ApiController
+    {
+        private readonly UsersOrchestrator _orchestrator;
+
+        public EmployerUserController(UsersOrchestrator orchestrator)
+        {
+            _orchestrator = orchestrator;
+        }
+
+        [System.Web.Http.Route("accounts", Name = "Accounts")]
+        [ApiAuthorize(Roles = "ReadUserAccounts")]
+        [System.Web.Http.HttpGet]
+        public async Task<IHttpActionResult> GetUserAccounts(string userRef)
+        {
+            return Ok(await _orchestrator.GetUserAccounts(userRef));
```

Comment:
This code has been slightly simplified from the original. The original orchestrator method `GetUserAccounts` always returned a hard-coded `HttpStatusCode.OK` in an `OrchestratorResponse`, so the orchestrator method was changed to return the old data payload (`ICollection<AccountDetailViewModel>`), and this controller action was simplified to hard-code an `OK` response.